.github/workflows/tests.yml

```
@@ -0,0 +1,41 @@
+name: UnitTests
+on: 
+  pull_request:
+    branch: 
+      - '*'
+  push:
+    branch:
+      - '*'
+
+jobs:
+  rpmlinter:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: rpmlinter
+      run:  make spec && docker run -v "$PWD"/python-boto.spec:/specs/python-boto.spec:z ubun/rpmlinter
+
+  python_test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [2.7,3.6]
```

наверное, стоит добавить еще питонов. и сделать это столбиком
```suggestion
        python-version:
          - 2.7
          - 3.6
          - 3.7
          - 3.8
          - 3.9
```